Game.java

```diff
@@ -3,51 +3,68 @@
 
 public class Game {
 
+    private final Scanner scanner;
     private char turn = 'o';
     private InputStream stream;
+    private Board board;
+    private String gameMode;
 
-    public Game(InputStream stream) {
+    public Game(InputStream stream, Board board) {
         this.stream = stream;
+        this.scanner = new Scanner(stream);
+        this.board = board;
     }
 
-    public void showBoard(Board board) {
-        System.out.println("|" + board.getState().substring(0, 3) + "|");
-        System.out.println("|" + board.getState().substring(3, 6) + "|");
-        System.out.println("|" + board.getState().substring(6, 9) + "|");
+    public void print(String text) {
+        System.out.println(text);
     }
 
-    public Boolean avaliablePos(Board board, int pos)  {
-        StringBuilder currentBoard = new StringBuilder(board.getState());
-        if (currentBoard.charAt(pos - 1) == '-') {
-            if (!(pos > 9) || !(pos < 1)) {
-                System.out.print("Must be a number from 1-9 \n");
-                return true;
-            }
-        }
-        return false;
+    public String showBoard(Board board) {
+        print("|" + board.getState().substring(0, 3) + "|\n|" + board.getState().substring(3, 6) + "|\n|" + board.getState().substring(6, 9) + "|");
+        return String.format("|" + board.getState().substring(0, 3) + "|\n|" + board.getState().substring(3, 6) + "|\n|" + board.getState().substring(6, 9) + "|");
```

Comment:
this method is dealing with a display/ui concern, but you may be just keeping this here as you haven't created a a Console class yet.